ibis/backends/tests/test_client.py

```diff
@@ -94,7 +104,18 @@ def test_create_table(backend, con, temp_table, lamduh, sch):
     )
 
     obj = lamduh(df)
-    con.create_table(temp_table, obj, schema=sch)
+    if backend.name() == "flink":
+        con.create_table(
+            temp_table,
+            obj,
+            schema=sch,
+            tbl_properties={
+                "connector": None,
+            },
+        )
+    else:
+        con.create_table(temp_table, obj, schema=sch)
+
```

Comment:
I know unifying `create_table` is on the radar, but I think we can clean this up a bit more, how about defining the extra `kwargs` in a dictionary that is otherwise empty and then passing those in to a single `create_table` call?

```suggestion
    kwargs = {}
    if backend.name() == "flink":
        kwargs = dict(tbl_properties={"connector": None})
    con.create_table(temp_table, obj, schema=sch, **kwargs)
```

Reply:
Did something similar!